src/plus/launchpad/launchpad.ts

```diff
@@ -978,20 +977,23 @@ export class LaunchpadCommand extends QuickCommand<State> {
 		state: StepState<State>,
 		context: Context,
 	): AsyncStepResultGenerator<{ connected: boolean | IntegrationId; resume: () => void }> {
-		const confirmations: (QuickPickItemOfT<IntegrationId> | DirectiveQuickPickItem)[] = [
-			createDirectiveQuickPickItem(Directive.Cancel, undefined, {
-				label: 'Launchpad prioritizes your pull requests to keep you focused and your team unblocked',
-				detail: 'Click to learn more about Launchpad',
-				iconPath: new ThemeIcon('rocket'),
-				onDidSelect: () =>
-					void executeCommand<OpenWalkthroughCommandArgs>(Commands.OpenWalkthrough, {
-						step: 'accelerate-pr-reviews',
-						source: 'launchpad',
-						detail: 'info',
+		const hasConnectedIntegration = some(context.connectedIntegrations.values(), c => c);
```

Comment:
Probably should filter to the supported integrations for Launchpad

Reply:
`context.connectedIntegrations` is already filtered to supported Launchpad integrations where it is set up. See `getConnectedIntegrations` of `LaunchpadProvider`